learn/advanced/faceted_search.md

```diff
@@ -0,0 +1,159 @@
+
+### Filters or facets
+
+In Meilisearch, facets are a specific use-case of filters. The question of whether something is a filter or a facet is mostly one pertaining to UX and UI design.
+
+Meilisearch does not differentiate between facets and filters. This means that, despite its name, facets can be used with any attributes added to `filterableAttributes`.
+
+## Configuring and using facets
+
+Like any other filter, attributes you want to use as facets must be added to the [`filterableAttributes`](/reference/api/settings.md#filterable-attributes) list in the index's settings before they can be used. Once they have been configured, you can search for facets with [the `facets` search parameter](/reference/api/search.md#facets).
+
+::: warning
+Synonyms don't apply to facets. Meaning, if you have `SF` and `San Francisco` set as synonyms, faceting by `SF` and `San Francisco` will show you different results.
+:::
+
+Suppose you have an ecommerce dataset called `ecommerce` containing the following fields:
+
+```json
+{
+  "id": "711decb2a3fdcbbe44755afc5af25e2f",
+  "title": "Kitchenex Stainless Steel Flatware Pie Server and Pie Cutter Set of 2",
+  "description": "| The Kitchenex Stainless Pie Server|…",
+  "url": "https://www.amazon.com/dp/B07D2LJFYW",
+  "images": [
+    "https://images-na.ssl-images-amazon.com/images/I/41AeR2oo75L._.jpg"
+  ],
+  "category": "Home & Kitchen",
+  "brand": "Dr. Pet",
+  "price": 16.84,
+  "rating": 4.7,
+  "reviews_count": 7,
+}
+```
+
+The following code sample allows you to create facets for the `brand` and `rating` attributes:
+
+<CodeSamples id="faceted_search_update_settings_1" />
+
+The response shows `pie server` products along with two new fields: `facetDistribution` and `facetStats`:
+
+```json
+{
+  "hits":[
+    …
+  ],
+  "query":"pie server",
+  "processingTimeMs":9,
+  "limit":20,
+  "offset":0,
+  "estimatedTotalHits":25,
+  "facetDistribution":{
+    "brand":{
+      "All Things Rae Dunn":1,
+      "BEERBONG .COM":1,
+      "Dr. Pet":1,
+      …
+      "senover":1
+    },
+    "rating":{
+      "0":7,
+      …
+      "4.8":1,
+      "5":3
+    },
+  },
+  "facetStats":{
+    "rating":{
+      "min":0.0,
+      "max":5.0
+    }
+  }
+}
+```
+
+### Facet distribution
+
+The `facetDistribution` object contains the number of matching documents distributed among the values of a given facet. The `facets` search parameter expects an array of strings. Each string is an attribute present in the `filterableAttributes` list.
+
+The following response shows the facet distribution when searching for `pie server`:
+
+```json
+{
+  …
+  "facetDistribution":{
+    "brand":{
+      "All Things Rae Dunn":1,
+      "BEERBONG .COM":1,
+      "Dr. Pet":1,
+      …
+      "senover":1
+    },
+    "rating":{
+      "0":7,
+      …
+      "4.8":1,
+      "5":3
+    }
+  }
+  …
+}
```

Comment:
If I were you, I would try to simplify this too. Perhaps showing the distribution for a single facet is better?